crates/solidity/testing/snapshots/cst_output/Keyword/while_keyword/input.sol

```diff
@@ -0,0 +1 @@
+while
```

Comment:
❤️ the new tests!
Please let me know if you have any feedback on that workflow.